src/NewProject/add_constraints_form.ui

```diff
@@ -86,6 +85,58 @@
          </property>
         </widget>
        </item>
+       <item>
+        <widget class="QGroupBox" name="groupBox">
+         <property name="title">
+          <string>Pin Location Assign Method:</string>
+         </property>
+         <layout class="QHBoxLayout" name="horizontalLayout_4">
+          <item>
+           <layout class="QHBoxLayout" name="horizontalLayout_3">
+            <item>
+             <widget class="QRadioButton" name="select_random">

```

Comment:
the selection in this GUI does nothing. it have to be connected with the option in the task table

Reply:
Resolved!